### PR TITLE
Brownfield land in wrong place jupyter notebook

### PR DESCRIPTION
**Pull request by CarlosCoelhoSL**
This is a notebook that produces a .csv containing information about brownfield land entities situated outside of their LPAs. It makes use of the data we have stored for the LPA polygons to filter brownfield land into entities outside their LPA, and then attempts to work out where the co-ordinates actually lie. 

For points outside their LPA, it firstly checks whether the point lies within England based on a polygon from https://cartographyvectors.com/map/1321-england-uk, and if it is, then it hits an API (https://postcodes.io/) with the co-ordinates to attempt to determine where in England it is. 
A maps link is generated so that you can instantly view the entity in question on google maps.
It also attempts to flip the sign of the co-ordinates and test whether this is within the LPA boundary to determine why this error might have occurred.

### CODE REVIEW

File: service_report/site-within-lpa-validator.ipynb [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+    "Note: This notebook makes use of a polygon of england when determining where a brownfield site is located. This polygon can be downloaded here: https://cartographyvectors.com/map/1321-england-uk, by selecting 'Choose Map Format' and then 'Well-Known-Text (WKT)'. The polygon is then stored in the same directory as this notebook."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "england_poly_json = \"englandjson.wkt\"\n",
```
Comment (ssadhu-sl) [inline service_report/site-within-lpa-validator.ipynb:16]: Can you consider keeping this file in the folder instead of the user downloading it?

File: service_report/site-within-lpa-validator.ipynb [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+   "source": [
+    "# Sort by distance:\n",
+    "df_brownfield_sites_outside_lpa_sorted = df_brownfield_sites_outside_lpa.sort_values(by=[\"Distance (Arbitrary Unit)\"], ascending=False)\n",
+    "\n",
+    "# Sort by organisation count:\n",
+    "# df_brownfield_sites_outside_lpa_counts = df_brownfield_sites_outside_lpa.organisation.value_counts()\n",
+    "# df_brownfield_sites_outside_lpa_sorted = df_brownfield_sites_outside_lpa.set_index(\"organisation\").loc[df_brownfield_sites_outside_lpa_counts.index].reset_index()\n",
+    "\n",
+    "df_brownfield_sites_outside_lpa_sorted = df_brownfield_sites_outside_lpa_sorted.reset_index(drop=True)\n",
+    "df_brownfield_sites_outside_lpa_sorted.to_csv(\"brownfield_sites_outside_lpa.csv\")\n",
```
Comment (ssadhu-sl) [inline service_report/site-within-lpa-validator.ipynb:213]: can we let the user know the file has been downloaded to their system?

**Review by ssadhu-sl — APPROVED** [review]
Looks very good and clean. I have approved but added some minor comments.
